react-restaurant-review-app/src/components/Login/Login.tsx

```diff
@@ -15,9 +16,20 @@ export default function Login() {
   function handleSubmit(event: React.FormEvent) {
     event.preventDefault();
 
-    console.log("Username:", formData.userName);
-    console.log("Password:", formData.password);
+    fetch("http://localhost:8080/login", {
```

Comment:
As a future improvement you can extract fetch to a separate file that would handle headers and json parsing, and then just call it in components by passing url, method and callback.